blob_server/models.py

```diff
@@ -35,15 +34,33 @@ class BlogPost(Base):
         secondary='post_tag_links'
     )
 
+    @property
+    def serializable(self) -> dict:
+        tags = [{'id': t.id, 'name': t.name} for t in self.tags]
+        return {
+            'id': self.id,
+            'created': self.created,
+            'content': self.content,
+            'tags': tags,
+        }
+
 
 class Tag(Base):
     """A tag indicating the subject of a blog post"""
     __tablename__ = 'tags'
-
     id = Column(Integer, primary_key=True)
     name = Column(String(32))
     posts = relationship(
         'BlogPost',
         back_populates='tags',
         secondary='post_tag_links'
     )
+
+    @property
+    def serializable(self) -> dict:
+        posts = [{'id': p.id, 'created': p.created, 'content': p.content} for p in self.posts]
```

Comment:
I would actually just include the 'id' in here. Reason being that for posts, it makes sense to send down the tag names, because you want to display them. But if I want to display information on a tag, I'll probably just want to be able to go get some number of posts with it, rather than having them all come down with the tag. So, the client only needs the "id," which will allow it to go get the posts if it wants to